src/app/apps/rup/mapa-camas/views/censos/censo-mensual/censo-mensual.component.html

```diff
@@ -23,7 +23,7 @@
                     <plex-button [disabled]="!fechaDesde || !fechaHasta || !selectedUnidadOranizativa"
                                  label="GENERAR CENSO" type="primary" (click)="generarCensoMensual()">
                     </plex-button>
-                    <plex-button [disabled]="!censo && requestInProgress" title="Descargar censo mensual" type="warning"
+                    <plex-button [disabled]="!censo || requestInProgress" title="(!censo) ? 'Debe generar el censo para descargarlo' : 'Descargar censo mensual'" type="warning"
```

Comment:
![image](https://user-images.githubusercontent.com/3813270/101898155-6753c180-3b8a-11eb-9aad-025ff442b90a.png)


```suggestion
                    <plex-button [disabled]="!censo || requestInProgress" [title]="(!censo) ? 'Debe generar el censo para descargarlo' : 'Descargar censo mensual'" type="warning"
```